examples/paint-mode/src/background.rs

```diff
@@ -11,9 +10,5 @@ pub fn get_background() -> Styled<Rectangle, PrimitiveStyle<Rgb888>> {
     let style = PrimitiveStyleBuilder::new()
         .fill_color(Rgb888::BLACK)
         .build();
-    Rectangle::new(
-        Point::new(0, 0),
-        Point::new(SCREEN_WIDTH as i32, SCREEN_HEIGHT as i32),
-    )
-    .into_styled(style)
+    Rectangle::new(Point::new(0, 0), Size::new(SCREEN_WIDTH, SCREEN_HEIGHT)).into_styled(style)
```

Comment:
This can just be `clear` https://docs.rs/embedded-graphics/0.7.1/embedded_graphics/draw_target/trait.DrawTarget.html#method.clear, I have another branch or two that optimizes this with a faster clear routine, but I believe it will behave the same as this code without any further modifications.